q2_feature_table/plugin_setup.py

```diff
@@ -71,7 +73,11 @@
                               'the resulting table.'),
         'axis': ('The axis to sample over. If "sample" then samples will be '
                  'randomly selected to be retained. If "feature" then '
-                 'a random set of features will be selected to be retained.')
+                 'a random set of features will be selected to be retained.'),
+        'seed': ('Set the seed for the subsampling. Using the same seed with '
+                 'the same table will always lead to the same result. Using '
+                 '"random", sets the seed to a random number. The random '
+                 'seed will not be logged in provenance.')
```

Comment:
Edits to the text. I'd leave the bit about provenance out, since we'll ideally be fixing that very shortly and it's common to all actions that take seeds right now. 

```suggestion
        'seed': ('Set the seed for the subsampling. Using the same seed with '
                 'the same table will always lead to the same result. By '
                 'default a random seed will be used.')
```